src/apis/data/endpoints.py

```diff
@@ -108,7 +109,13 @@ class GetStateCasesTotals(Resource):
     @data_endpoints.marshal_with(sum_all_cases_response)
     def get(self):
         """Cases per state report"""
-        return state_services.get_sum_state_cases()
+        response = state_services.get_sum_state_cases()
+        if response['totalCases'] is None and \
+                response['totalCasesMS'] is None and \
+                response['deaths'] is None:
+            abort(404, "No report found for states")
```

Comment:
Maybe you could use [`all()`](https://www.programiz.com/python-programming/methods/built-in/all) here:

```suggestion
        if not all(report for report in response.values()):
            abort(404, "No report found for states")
```